imagersite/imagerprofile/templates/base.html

```diff
@@ -1,15 +1,27 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>Imager Site</title>
-  </head>
-  <body>
-    <header>
-      <h1>Imager Site</h1>
-    </header>
-    {% block main %}{% endblock %}
-    <footer>
-      {% block nav %}{% endblock %}
-    </footer>
-  </body>
+<head>
+<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
```

Comment:
bootstrap has both css and javascript.  Use them both, and then start with a template from the bootstrap site.  It'll help you to understand how to use HTML that fits bootstrap's expectations.